### DISCUSSION TITLE
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xa4 in position 9: invalid start byte

### DISCUSSION THREAD

**Discussion by tt960922** [openai/whisper#302] [Q&A] [answered]
import whisper

model = whisper.load_model("base")

result = model.transcribe(r"F:\whisper\test2.wav")

print(result["text"])

I don't understand why I keep getting this error

    raise RuntimeError(f"Failed to load audio: {e.stderr.decode()}") from e
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xa4 in position 9: invalid start byte

**Comment by jongwook** [comment, marked as answer]
It appears that the code failed to load the audio file for some reason and even failed to display that error because `e.stderr` did not contain a valid UTF-8 string. Can you load the audio file using [`wave`](https://docs.python.org/3/library/wave.html) or [`scipy.io.wavfile.read`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.io.wavfile.read.html)?

**Comment by 997042353** [comment]
Did you solve it?
I had the same problem today

**Comment by Jieszs** [comment]
I had the same problem when I use Windows system,and now I solve it.
The reason is i didn't install ffmpeg.
`pip install ffmpeg-python `

**Reply by Fengxiaoyu14** [reply]
That's correct, but pip install doesn't work for me. 
I installed ffmpeg manually and added directory to PATH, and it worked.
